orchestrator/tests/test_board_task_handlers.py

```
@@ -499,3 +505,825 @@ def test_sse_stream_gated_by_tasks_read_scope():
+
+# ---------------------------------------------------------------------------
+# P224-RVW-4 -- the WRITE layer (create/assign) resolves an agent NAME against
+# ACTIVE agents and REFUSES an ambiguous name instead of silently .first()-ing
+# a Postgres-row-order pick. Agent.name has no unique constraint (only
+# (workspace_id, slug)), so 'Atlas'/'atlas' can coexist; and the classifier
+# matches active-only while the old write query matched any status — the two
+# rosters could diverge and dispatch the wrong (or a deactivated) agent.
+# ---------------------------------------------------------------------------
+
+
+def _ag(id, name, status="active"):
```

Ruff (0.16.2)

[error] Function argument `id` is shadowing a Python builtin

(A002)